src/developer/web-api/data-exchange.md

```diff
@@ -39,6 +39,20 @@ It is recommended to either specify basic authentication or PAT authentication.
 
 Note that PAT support was introduced in version 2.38.1, which means that in order to use PAT authentication, the target DHIS 2 instance must be version 2.38.1 or later.
 
+### Sharing
+Like other metadata objects, fine-grained security can be associated with aggregate data exchanges. Each exchange can be shared with individual users and/or user groups to control which users have access to the specific exchange. External data exchanges contain authentication details of users on the target system, thus great care should be
+taken to ensure that only authorized users have access to actually submit data which results from the exchange.
+
+The following table summarizes how sharing can be used with aggregate data exchanges.
+
+
+| Sharing | Effective permissions                                                              |
+| -------- |-----------------------------------------------------------------------------------|
+| "r-------" | Can view metadata of the data exchange. |
+| "-w------" | Can edit metadata of the data exchange. |
+| "--r-----" | Can view data which is part of the exchange. |
```

Comment:
Correct me if i'm wrong. Data Read is not implemented in AggregateDataExchange level. Instead, it is controlled by sharing on the individual metadata objects (data elements, indicators) which are part of the request itself. The sharing on the individual “dx” objects control whether the user has access to the data or not.